setup.py

```diff
@@ -37,7 +37,7 @@
         'importlib_metadata',
         'lxml',
         'numpy',
-        'scipy',
+        'scipy', 'requests', 'boto3', 'urllib3'
```

Comment:
These should be on their own line and sorted -- if/when we start adding restrictions it will be a lot clearer that way